test/helper.spec.js

```diff
@@ -64,7 +64,7 @@ describe('runner', () => {
             await templ(bigodon);
             const deltaT = Date.now() - start;
             expect(deltaT).to.be.at.least(49);
```

Comment:
I would be better to change this and the two wait helpers above to 10x too (this 49 becomes 490, `{{wait 20}}` becomes `{{wait 200}}`, and so on). That way, the limit will also fail for longer than expected waits

Reply:
I'm sorry for that, i have change it. Thanks for the feedback 😄